python/ray/ml/train/integrations/torch/torch_trainer.py

```diff
@@ -0,0 +1,54 @@
+from typing import Callable, Optional, Dict
+
+from ray.train.torch import TorchConfig
+from ray.ml.trainer import GenDataset
+from ray.ml.train.impl.data_parallel_trainer import DataParallelFunctionTrainer
+from ray.ml.config import ScalingConfig, RunConfig
+from ray.ml.preprocessor import Preprocessor
+from ray.ml.checkpoint import Checkpoint
+from ray.util import PublicAPI
+
+
+@PublicAPI(stability="alpha")
+class TorchTrainer(DataParallelFunctionTrainer):
+    """A Trainer for data parallel PyTorch training.
+
+    This Trainer runs the provided function on multiple Ray Actors. The
+    provided datasets are automatically sharded to each Ray actor running
```

Comment:
Can we say just the train dataset is sharded?

Reply:
Updated